### DISCUSSION TITLE
Google

### DISCUSSION THREAD

**Discussion by Tnixc** [catppuccin/catppuccin#1837] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Google. Yes

**Additional context 📷 (optional)**
![image](https://user-images.githubusercontent.com/85466117/211143294-7e4f7875-53bd-4409-b12b-581ffa0e81c2.png)
![image](https://user-images.githubusercontent.com/85466117/211143311-b917f741-95d8-4ad2-b300-a35502166c97.png)
Needs some fixing up

**Comment by sgoudham** [comment]
Looks great! Is there a repository anywhere with the WIP code?

**Comment by mustafa-archived** [comment]
Hi, this seems unmaintained so I thought it would be cool if I make one https://github.com/mustafakhalaf-git/google

**Comment by mustafa-archived** [comment]
@sgoudham can you review my work so far? I would like to finish this soon

**Reply by sgoudham** [reply]
Will do! I've been pretty busy but I'll get around to it.

**Comment by Tnixc** [comment]
Can't believe someone actually picked this up lol, good work!

**Comment by vanillyn** [comment]
The buttons at the top are the same between the themes:
![image](https://user-images.githubusercontent.com/108785622/232605300-ae0a01b9-7fa7-44e5-b7cc-cb85c05df434.png)
![image](https://user-images.githubusercontent.com/108785622/232605339-3d8046d8-ffcd-4622-afc4-f36ad9fae42c.png)

Google Image Search is unthemed:
![image](https://user-images.githubusercontent.com/108785622/232605420-3375c767-dd38-4711-9461-41ea92cd5ca1.png)

All of Travel is unthemed
![image](https://user-images.githubusercontent.com/108785622/232605632-bcc2f381-5f12-4fb3-ad3c-cddb7dd4b889.png)

**Reply by mustafa-archived** [reply]
Stuff will be limited to search only (for now)

Image upload is fixed, not sure where the buttons you're talking about are

**Comment by stevedylandev** [comment]
If there's a way to scope this only to google search I think it would be ready to go, but since it does spill into other google apps it might need more work

**Reply by stevedylandev** [reply]
Example 

<img width="578" alt="Screenshot-Arc-04-17-2023-16-17@2x" src="https://user-images.githubusercontent.com/73185218/232605973-6544c2ce-562f-4873-a163-1e876a1c3682.png">

**Reply by mustafa-archived** [reply]
Fixed

**Comment by kkrypt0nn** [comment]
Overall amazing theme and really like it :D

Here are some small things I've noticed while testing:

<details>
<summary>Previous Searches</summary>

<img width="636" alt="image" src="https://user-images.githubusercontent.com/43011723/232605890-6de51122-7f9e-465a-974f-ad8cbd528537.png">

<img width="857" alt="image" src="https://user-images.githubusercontent.com/43011723/232607994-101e8a62-bf65-4b5f-b43b-47aa1f76533a.png">

Previous searches's text color is not styled

</details>

<details>
<summary>News</summary>

<img width="736" alt="image" src="https://user-images.githubusercontent.com/43011723/232606194-c3585617-2e27-4955-b846-340bb12989b3.png">

Cards background in the news section are not styled

</details>

<details>
<summary>Flights</summary>

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/43011723/232606361-96a0ade1-9d52-45d4-9fca-344278e71995.png">

Flights cards & footer are not styled

</details>

<details>
<summary>Border of Video Badges</summary>

<img width="789" alt="image" src="https://user-images.githubusercontent.com/43011723/232606705-bf09498b-a01d-41cb-bbc3-36553bb97edc.png">

The rounded border around these badges seem to not be styled as well

</details>

<details>
<summary>Tools > Selected Filter</summary>

<img width="729" alt="image" src="https://user-images.githubusercontent.com/43011723/232607412-fb56c334-7e6b-4c0c-8d44-e95bf00c64a3.png">

When enabling tools and clicking in a filter, the colors of the selected filter is not styled

</details>

<details>
<summary>Quick Settings</summary>

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/43011723/232607633-9d96b958-344e-4a8f-9620-1f18ba08aa4b.png">

Clicking on the cog icon to the left of the profile avatar shows the quick settings unstyled

</details>

<details>
<summary>Profile Selection</summary>

<img width="387" alt="image" src="https://user-images.githubusercontent.com/43011723/232607824-80513a2f-b791-4c6f-9a01-d6bf2bfa91ca.png">

Clicking on the profile avatar shows the unstyled account switcher

</details>

<details>
<summary>App Choice</summary>

<img width="339" alt="image" src="https://user-images.githubusercontent.com/43011723/232608104-4b5769c4-4c34-4052-b18e-ec966f059984.png">

Clicking on the four squares to choose an app to go to shows unstyled window

</details>

<details>
<summary>Search By Image</summary>

<img width="651" alt="image" src="https://user-images.githubusercontent.com/43011723/232608239-031dd79b-e031-420c-a4a1-c92d6ebf7ca8.png">

Clicking on the camera icon to search by image URL/file shows an unstyled window

</details>

<details>
<summary>Books *"Found inside"*</summary>

<img width="728" alt="image" src="https://user-images.githubusercontent.com/43011723/232608622-197ec7ac-579a-4c9f-a377-25a648970ca0.png">

The "Found inside" text color seem to be unstyled

</details>

<details>
<summary>Finance</summary>

<img width="1249" alt="image" src="https://user-images.githubusercontent.com/43011723/232608863-50adca23-29c0-4cc4-8d3d-6c529eac1d90.png">

Entire finance page seems to be unstyled - though I've put that one in italic because it seems like Google themselves can't get this show in dark mode either

</details>

**Reply by kkrypt0nn** [reply]
May be worth focusing on the search page only at the moment - though could be kept for the future ^^

**Reply by mustafa-archived** [reply]
Previous Search: Done ✅
Border of video badges: Done ✅
Tools > Selected Filter: Done ✅
Quick Settings: Done ✅
Profile Selection & App Choice: Done ✅
Search by Image: Done ✅
Found Inside: Done ✅

Flights: Non-search won't fix ⛔
Finance: Non-search won't fix ⛔
News: Doesn't appear won't fix ⛔

**Reply by mustafa-archived** [reply]
@kkrypt0nn hey so I'm facing an issue, I cannot find the 'news' thing in the same form it appears for you, any idea how I can make it appear?

**Reply by kkrypt0nn** [reply]
@mustafakhalaf-git unfortunately not, for me it always looked like that for the past few years - no idea if it's a setting to enable or something similar.

Sorry 😔

**Reply by mustafa-archived** [reply]
@kkrypt0nn welp, guess I won't be fixing it until they show up

**Reply by mustafa-archived** [reply]
@kkrypt0nn Finished all of them and updated the css, you may check them if you want 

btw you need to have iframe setting enabled in stylus options

**Comment by samnystrom** [comment]
The button to view more image tags seems to be unthemed.
![6423FCDC-A4E6-4D5E-8970-3F2CA25E7BAC](https://user-images.githubusercontent.com/82178396/232626515-ec32e1b9-83a3-4885-9ac8-67c7430e7584.jpeg)

**Reply by mustafa-archived** [reply]
Will check it tomorrow hopefully 

Edit: Fixed

**Comment by GenShibe** [comment]
The main thing that needs to be tweaked is the fact that you have to go into the quick settings menu and switch to light mode for latte to look best, as well as the lack of styling for the more images tag; 

one thing that could be worth looking into would be to theme the buttons that say news/videos/shopping into the background color and when it's active, have the main color of the button be the chosen accent color

![sadjkl](https://user-images.githubusercontent.com/29279972/232654367-5f0d4856-be38-4bec-be91-62d507d6a68c.png)
![234w](https://user-images.githubusercontent.com/29279972/232654381-e8a23fbf-ac1b-4ae5-be31-4b7e58a79f66.png)
![14](https://user-images.githubusercontent.com/29279972/232654393-2b869937-f6c3-4c15-b404-c962c9ff1bc6.png)
![ads](https://user-images.githubusercontent.com/29279972/232654403-e2b490b6-d010-4a98-a6cf-8a98ca835237.png)

**Reply by mustafa-archived** [reply]
The issue is mainly with the shadows that appear under boxes, in dark mode they are dark colored so in latte it appears dark, maybe I should add a note in the readme to switch to light mode when going on latte

**Reply by mustafa-archived** [reply]
Talking about the buttons, they look different on my side, not circled, is there an option to enable to get them there

**Reply by mustafa-archived** [reply]
This is how they look from my side (they are themed but screenshot taken on phone so no color in it)
![Screenshot_20230419-212238_Fennec](https://user-images.githubusercontent.com/42213155/233165843-bb9dcd2b-84b4-4a3b-9c26-37ecf87000a9.jpg)

**Comment by MAHcodes** [comment]
looks great, thank you for your effort! I noticed this is unthemed
![image](https://user-images.githubusercontent.com/69465962/232660511-540e31b5-507c-4ccc-a44b-8647c0c94e55.png)

**Reply by mustafa-archived** [reply]
Fixed

**Comment by mustafa-archived** [comment]
Updated the css

gonna mention my friend here @thismoon

**Comment by sgoudham** [comment]
This thread is getting a little hard to scroll through, could I ask that you try to keep your replies contained within the original comment thread instead of creating a new comment each time? @mustafakhalaf-git Thanks!

**Reply by mustafa-archived** [reply]
trying my best to do so

**Reply by mustafa-archived** [reply]
@sgoudham think I've finished pretty much everything, now I'll just wait until it's reviewed

**Comment by Tnixc** [comment]
Looking great, but there's some random ass buttons here that haven't been themed:
![image](https://user-images.githubusercontent.com/85466117/233658248-3f8cac94-b1b1-43f7-b4e7-d4790465a150.png)

**Reply by mustafa-archived** [reply]
Thanks, can you tell me where exactly is that though?

**Reply by Tnixc** [reply]
click on an image and scroll down, it's in the side panel.

**Reply by mustafa-archived** [reply]
@Tnixc should be fixed now https://github.com/mustafakhalaf-git/google/commit/a19a229ce05d365764660a10161b8c0303c8be56

**Comment by mustafa-archived** [comment]
Transferred to https://github.com/catppuccin/catppuccin/issues/2034
